docs/src/inverse_problems/petab_ode_param_fitting.md

```diff
@@ -0,0 +1,487 @@
+```petab1
+using DataFrames
+measurements = DataFrame(obs_id="obs_P", time=sol.t[10:10:end], measurement=data)
+```
+Since, in our example, all measurements are of the same observable, we can set `obs_id="obs_P"`. If several different observables were measured, `obs_id` would be a vector of the same length as `time` and `measurement`.
+
+### Creating a PEtabModel
+Finally, we combine all inputs in a single `PEtabModel`. To it, we also pass the initial conditions of our simulations (using the `state_map` argument). It is also possible to have [initial conditions with uncertainty](@ref petab_simulation_initial_conditions_uncertainty), [that vary between different simulations](@ref petab_simulation_conditions), or [that we attempt to fit to the data](@ref petab_simulation_initial_conditions_fitted).
+```petab1
+petab_model = PEtabModel(rn, observables, measurements, params; state_map=u0)
+nothing # hide
+```
+
+### Fitting parameters
+We are now able to fit our model to the data. First, we create a `PEtabODEProblem`. Here, we use `petab_model` as the only input, but it is also possible to set various [numeric solver and automatic differentiation option](@ref petab_simulation_conditions) (such as method or tolerance).
+```petab1
+petab_problem = PEtabODEProblem(petab_model)
+nothing # hide
+```
+To fit a parameter set we use the `calibrate_model` function. In addition to our `PEtabODEProblem`, we must also provide an initial guess (which can be generated with the `generate_startguesses` function) and an optimisation algorithm (which needs to be imported specifically). PEtab.jl supports various [optimisation methods and options](@ref petab_optimisation_optimisers). 
+```petab1
+using Optim
+p0 = generate_startguesses(petab_problem, 1)
+res = calibrate_model(petab_problem, p0, IPNewton())
+```
+
+We can now simulate our model for the fitted parameter set, and compare the result to the measurements and true solution.
+```petab1
+oprob = ODEProblem(rn, u0, (0.0, 10.0), get_ps(res, petab_problem))
+
+sol = solve(oprob, Tsit5(); saveat=0.1)
+plot!(sol; idxs=4, label="Fitted solution")
+```
+Here we use the `get_ps` function to retrieve a full parameter set using the optimal parameters (alternatively, the `ODEProblem` or fitted simulation can be retrieved directly using the `get_odeproblem` or `get_odesol` [functions](https://sebapersson.github.io/PEtab.jl/dev/API_choosen/#PEtab.get_odeproblem), respectively). The calibration result can also be found in `res.xmin`, however, note that PEtab automatically ([unless a linear scale is selected](petab_parameters_scales)) converts parameters to logarithmic scale, so typically `10 .^res.xmin` are the values of interest. If you investigate the result from this example you might note that the found parameter set (while it fits the data well) does not correspond to the true parameter set. This phenomenon is related to *identifiability*, and is very important for parameter fitting.
```

Comment:
Here I also think you should mention the `get_u0` function, because if initial values are estimated then in order to get the correct simulations the correct initial values are also needed (this is automatically taken care of by `get_odeproblem` and `get_odesol`

Reply:
added